projects/microphysics/scripts/prognostic_run.py

```diff
@@ -48,6 +42,7 @@ def get_env(args):
 )
 parser.add_argument("--segments", "-n", type=int, default=1, help="number of segments")
 parser.add_argument("--wandb-project", default="microphysics-emulation")
+parser.add_argument("--job-type", default="prognostic_run")
```

Comment:
Maybe cleaner to use the `WANDB_JOB_TYPE` environment variable instead of adding the arg: https://docs.wandb.ai/guides/track/advanced/environment-variables

Reply:
If you prefer the arg, then I would name it similarly to `--wandb-project` to clarify: 

```suggestion
parser.add_argument("--wandb-job-type", default="prognostic_run")
```

Reply:
Chose the argument since the `.init(job_type=...)` will override the environment variable.  This way it's nicely recorded as the command that invoked wandb.